src/less/mixins/public/skeleton-mixins.less

```diff
@@ -0,0 +1,15 @@
+:root {
+    --skeleton-border-radius: 0;
+    --skeleton-height: 48px;
+    --skeleton-width: 100%;
+    --skeleton-background: var(--color-loading-fill);
+}
+
+.skeleton-base(@border-radius: var(--skeleton-border-radius), @height: var(--skeleton-height), @width: var(--skeleton-width) ) {
+    background: var(--skeleton-background);
```

Comment:
For consistency, we should add a custom variable. 
So like
`.backgroundColorToken(skeleton-background-color, current-skeleton-backround);`
And then rename `--skeleton-background` to `--current-skeleton-backround`
The skeleton-background-color is not used yet, but when we create themes and such, then it will be used.

Reply:
Done.